woocommerce/woocommerce-android
# Add toolbar for CustomerOrderNoteEditingFragment

Part of #10570 

### Description

This PR introduces individual toolbars for Customer order note editing screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Customer order note editing (Order list -> Order detail -> Add customer note) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="431" alt="Screenshot 2024-02-12 at 5 39 47 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/1f213150-44b6-4695-aa29-1eee7978bbef">
